src/components/WeeklyCalendar/helpers/isToday.js

```diff
@@ -0,0 +1,16 @@
+export default function isToday(day) {
+    try {
```

Comment:
can you explain why we are using the try-catch, the day passed in internal implementation should be normalized at the beginning which guarantees the value is valid across all the implementations if it comes wrong from the consumer we should have fallback values